python/pyspark/worker.py

```diff
@@ -111,8 +111,25 @@ def verify_result_length(result, length):
                                      map(verify_result_type, f(*iterator)))
 
 
-def wrap_grouped_map_pandas_udf(f, return_type, argspec):
+def wrap_cogrouped_map_pandas_udf(f, return_type):
 
+    def wrapped(left, right):
```

Comment:
Is `left` a list of pd.Series here? Probably name them `left_series` and `value_series` to be more readable?

Reply:
yes they are-  they are value series for left and right sides of the cogroup respectively. Agreed that the names aren't the best. I'll  improve them when I do a tidy up.